widgets/Ariba-connector.json

```diff
@@ -0,0 +1,54 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "display-name": "Ariba",
+  "configuration-groups": [
+    {
+      "label": "Credentials-updated",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Ariba Client ID",
+          "name": "clientId",
+          "widget-attributes": {
+            "placeholder": "Ariba Client ID"
+          }
+        },
+        {
+          "widget-type": "password",
+          "label": "Ariba Client Secret",
+          "name": "clientSecret",
+          "widget-attributes": {
+            "placeholder": "Ariba Client Secret"
+          }
+        },
+        {
+          "widget-type": "password",
+          "label": "Ariba API Key",
+          "name": "apiKey",
+          "widget-attributes": {
+            "placeholder": "Use any Application Key of an Application(s) you created in developer portal."
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "API Endpoint",
+          "name": "baseURL",
+          "widget-attributes": {
+            "placeholder": "Base path of the Ariba API, for example, https://sandbox.api.sap.com"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Realm",
+          "name": "realm",
+          "widget-attributes": {
+            "placeholder": "Realm name from which the data is to be extracted, for example, S4VALL."
+          }
+        }
+      ]
+    }
```

Comment:
Add System Type as a radio button field with values prod and sandbox as we were doing previously.

Reply:
Done